app/ports/views.py

```diff
@@ -252,12 +252,20 @@ def all_builds_filter(request):
 
 def stats(request):
     days = request.GET.get('days', 30)
+    days_ago = request.GET.get('days_ago', 0)
+
+    # Validate days and days_ago
+    for value in days, days_ago:
+        check, message = validate_stats_days(value)
```

Comment:
Does this mean that duration of 0 days is valid?

(I don't care that much though ... I wouldn't mind if this wasn't so strict anyway.)

Reply:
Yeah, but the selectors in the template don't allow users to actually choose `0`.